docs/sql-ref-ansi-compliance.md

```diff
@@ -193,7 +193,7 @@ Below is a list of all the keywords in Spark SQL.
 |DIRECTORY|non-reserved|non-reserved|non-reserved|
 |DISTINCT|reserved|non-reserved|reserved|
 |DISTRIBUTE|non-reserved|non-reserved|non-reserved|
-|DIV|non-reserved|non-reserved|non-reserved|
+|DIV|non-reserved|non-reserved|not a keyword|
```

Comment:
Ah, I see. We cannot remove this case: `non-reserved|non-reserved|not a keyword`? I think users don't need to care about the words.

Reply:
But is DIV a keyword?

Reply:
`DIV` is a non-reserved keyword in Apache Spark. I guess we can have this.